src/Cache/__tests__/Cache-test.js

```diff
@@ -266,11 +268,10 @@ describe('Cache', () => {
         Promise.resolve('baz value')
       );
 
-      // jest has some trouble with promises and timeouts within promises :(
-      jest.runAllTimers();
-      jest.runAllTimers();
-
-      expect(fs.writeFile).toBeCalled();
+      setTimeout(() => {
```

Comment:
Yeah I guess I can look into this as a follow-up as well, I don't wanna block you on this test.